requirements.txt

```diff
@@ -3,5 +3,5 @@ mock
 nose
 six
 requests>=2.0.0
-PyJWT==1.7.1
-twine
\ No newline at end of file
+PyJWT>=2.0.0
```

Comment:
I'd recommend putting a high bar on this dependency. There might be a point in the future where it will become 3.x and will break API.

Reply:
Good spot, updated. Thanks!